### PR TITLE
View change with f replicas collected stable checkpoint

### PR DESCRIPTION
**Pull request by HristoStaykov**
This test is a recreation of a infinite View Change scenario, where we have only F Replicas that have gathered a Stable Checkpoint. This causes disagreement in the computed Restrictions for the previous View's Working Window and View Change is never finalized.

### CODE REVIEW

**Comment by f-squirrel** [comment]
@HristoStaykov , what an amazing way to document issues that are hard to reproduce!

**Review by teoparvanov — APPROVED** [review]
Awesome, thanks @HristoStaykov !

File: tests/apollo/test_skvbc_chaotic_startup.py [hunk 1]
```diff
@@ -48,6 +50,99 @@ class SkvbcChaoticStartupTest(unittest.TestCase):
 
     __test__ = False  # so that PyTest ignores this test scenario
 
+    @unittest.skip("Edge case scenario - not part of CI")
```
Comment (teoparvanov) [inline tests/apollo/test_skvbc_chaotic_startup.py:53]: can you also specify the BC-XXXX ticket number that this test allows to reproduce?

Comment (f-squirrel) [reply tests/apollo/test_skvbc_chaotic_startup.py:53]: AFAIK, the approach is to avoid mentioning ticket numbers since they are not public.
